src/test/statistics/chunk_statistics/range_filter_test.cpp

```diff
@@ -1,278 +0,0 @@
-#include <functional>
-#include <memory>
-#include <string>
-#include <utility>
-#include <vector>
-
-#include "base_test.hpp"
-#include "gtest/gtest.h"
-
-#include "utils/assert.hpp"
-
-#include "statistics/chunk_statistics/range_filter.hpp"
-#include "types.hpp"
-
-namespace opossum {
-
-template <typename T>
-class RangeFilterTest : public ::testing::Test {
- protected:
-  void SetUp() override {
-    // Manually created vector. Largest exlusive gap (only gap when gap_count == 1) will
```

Comment:
Why removed?

Reply:
There is another version in `statistics_objects/range_filter_test.cpp`. CMake doesn't even know about this one.